fiducial_slam/src/map.cpp

```diff
@@ -80,6 +80,7 @@ static double updateVarianceDavid(const tf2::Vector3 &newMean,
     return newVar;
 }
 
+#if 0
```

Comment:
Just comment out or remove? 

At least explain why it is disabled

Reply:
It's not used and generates a warning because it's static.   We could make it non-static or remove it.